### ISSUE TITLE
Home path is repeated twice when trying to init fleek

### ISSUE THREAD

**Issue by 1024bees** [ublue-os/fleek#221]
**Describe the bug**
`fleek init --force` fails with the following output: 

``` bash 
fleek init --force                 
 [*]  Initialize fleek
 [i]  Writing configuration files
 [i]  Bling Level: default
 [i]  Enter User Details for Git Configuration:
 [i]  Detected your name: James
Use detected name: James Yes
Use detected email: jimmy@1024bees.com Yes
Choose Git SSH Key id_rsa.pub
  ERROR   
  ERROR   
          Error: Creating configuration files
          source: open /home/james/home/james/.local/share/fleek/.fleek.yml: no such file or directory
  ERROR   

```

I've encountered this when  I tried to upgrade fleek today to pick up the fix for #200. I had issues and decided to try to blow away everything and start anew, so I moved my `~/.local/share/fleek` folder to safe place, removed ~/.fleek.yml and ran `fleek init --force`

**To Reproduce**
Run `fleek init --force` with a fresh fleek 

**Expected behavior**

**Environment**
 - OS: [e.g. macOS Mojave, Ubuntu 18.04, ...] Ubuntu 20.04
 - Shell: [e.g. bash, zsh, ...] zsh
 - fleek version [e.g. 0.4.1] This fails for both 0.9.13 and 0.9.14 -- I picked up 0.9.14 after seeing #220 being added. 

**Additional context**
I've checked that my $HOME is set to `/home/james`

**Comment by stationaryStation** [comment]
I am also having the same issue on Ublue Bluefin (Fedora 38)

Can't apply configuration using `fleek apply`

**Comment by 1024bees** [comment]
I think it was introduced recently -- I have a fix on #222 if you'd like to try it out
